src/test/java/com/wikia/webdriver/testcases/visualeditor/VEImageTests.java

```diff
@@ -0,0 +1,104 @@
+
+  @Test(groups = {"VEImageTests", "VEMediaSetting"})
+  @Execute(asUser = User.USER_9)
+  public void editImageCaption() {
+    String captionText = "test123";
+
+    VisualEditorPageObject ve =
+        new VisualEditorPageObject(driver).openVEOnArticle(wikiURL, PageContent.ARTICLE_NAME_PREFIX
+                                                                    + DateTime.now().getMillis());
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = ve.clickImageButton();
+    mediaDialog = mediaDialog.searchMedia("h");
+    ve = mediaDialog.addExistingMedia(1);
+    ve.verifyVideos(1);
+    ve.selectMedia();
+    VisualEditorMediaSettingsDialog mediaSettingsDialog = ve.openMediaSettings();
+    mediaSettingsDialog.typeCaption(captionText);
+    ve = mediaSettingsDialog.clickApplyChangesButton();
+    ve.verifyVideoCaption(captionText);
+    VisualEditorSaveChangesDialog save = ve.clickPublishButton();
+    ArticlePageObject article = save.savePage();
+    article.verifyVEPublishComplete();
+  }
+
+  @Test(groups = {"VEImageTests", "VEMediaSetting"})
+  @Execute(asUser = User.USER_9)
+  public void changeImageAlignment() {
+    int numOfMedia = 3;
+    List<String> wikiTexts = new ArrayList<String>();
+    wikiTexts.add("|centre");
+    wikiTexts.add("|left");
+
+    String randomArticleName =
+        PageContent.ARTICLE_NAME_PREFIX + new VisualEditorPageObject(driver).getTimeStamp();
+    VisualEditorPageObject ve =
+        new VisualEditorPageObject(driver).openVEOnArticle(wikiURL, randomArticleName);
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = ve.clickImageButton();
+    mediaDialog = mediaDialog.searchMedia("h");
+    ve = mediaDialog.addExistingMedia(numOfMedia);
+    ve.verifyMedias(numOfMedia);
+    ve.verifyEditorSurfacePresent();
+    ve.verifyVEToolBarPresent();
+    ve.selectMediaByIndex(2);
+    VisualEditorMediaSettingsDialog mediaSettingsDialog = ve.openMediaSettings();
+    mediaSettingsDialog.selectSettings(Setting.ADVANCED);
+    mediaSettingsDialog.clickAlignment(Alignment.LEFT);
+    ve = mediaSettingsDialog.clickApplyChangesButton();
+    ve.verifyEditorSurfacePresent();
+    ve.verifyVEToolBarPresent();
+    ve.selectMediaByIndex(0);
+    mediaSettingsDialog = ve.openMediaSettings();
+    mediaSettingsDialog.selectSettings(Setting.ADVANCED);
+    mediaSettingsDialog.clickAlignment(Alignment.CENTER);
+    ve = mediaSettingsDialog.clickApplyChangesButton();
+    ve.verifyEditorSurfacePresent();
+    ve.verifyVEToolBarPresent();
+    VisualEditorSaveChangesDialog saveDialog = ve.clickPublishButton();
+    VisualEditorReviewChangesDialog reviewDialog = saveDialog.clickReviewYourChanges();
+    reviewDialog.verifyAddedDiffs(wikiTexts);
+    saveDialog = reviewDialog.clickReturnToSaveFormButton();
+    ArticlePageObject article = saveDialog.savePage();
+    article.verifyVEPublishComplete();
```

Comment:
I would try to separate the code on the meaningful pieces (small methods which will tell me what is going on from the method name).